src/components/Category/Category.js

```diff
@@ -0,0 +1,9 @@
+function Category() {
+  return (
+    <div>
+      <h1>Coming Soon</h1>
+    </div>
```

Comment:
Please ensure that the `Category component` displays only a button with text "Check status according to the project requirement.